09 Redux/readme.md

```diff
@@ -234,7 +234,7 @@ const createEmptyMember = (): MemberEntity => ({
   avatar_url: '',
 });
 
-export const memberReducer = (state = createEmptyMember(), action) => {
```

Comment:
This is `memberReducer`. 

In line [272](https://github.com/Lemoncode/react-typescript-samples/pull/109/files#diff-147d379a8938053d9e4f4be2930acb43R272) is the typo. Could you change it?